### ISSUE TITLE
SyntaxError in monitor.rb under Rails 4.2

### ISSUE THREAD

**Issue by andrewsage** [lbeder/health-monitor-rails#56]
I'm running Rails 4.2 and get the following error:

```
/health-monitor-rails-8.6.1/lib/health_monitor/monitor.rb:48: syntax error, unexpected '.' (SyntaxError)
    configuration.error_callback&.call(e)
```

If I switch to using version 8.6.0 of the gem then the error does not occur.

**Comment by lbeder** [comment]
Can you check which Ruby version are you using? `&.` is Ruby 2.3.0's new safe navigation operator and it should've been supported by Ruby >2.3.0. I can make it compatible with an older Ruby version, but its support is a very low priority.

**Comment by lbeder** [comment]
I'm fixing it in https://github.com/lbeder/health-monitor-rails/pull/57 and will release soon as 8.6.2. Thanks for reporting it!

**Comment by andrewsage** [comment]
The app is using Ruby 2.2.2.

**Comment by lbeder** [comment]
Fixed and released as 8.6.2

**Comment by andrewsage** [comment]
Thank you that has fixed that issue for me.
